src/fastoad_cs25/models/geometry/geom_components/wing/components/compute_center_chord.py

```diff
@@ -0,0 +1,93 @@
+"""
+Estimation of center wing chord (l1')
+"""
+#  This file is part of FAST-OAD : A framework for rapid Overall Aircraft Design
+#  Copyright (C) 2025 ONERA & ISAE-SUPAERO
+#  FAST is free software: you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation, either version 3 of the License, or
+#  (at your option) any later version.
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#  You should have received a copy of the GNU General Public License
+#  along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+import numpy as np
+import openmdao.api as om
+from scipy.interpolate import interp1d
+
+
+class ComputeCenterChord(om.ExplicitComponent):
+    # TODO: Document equations. Cite sources
```

Comment:
If it is alright with you, we have started quotting the document that provide this equation in our docstrings in FAST-GA, see: https://github.com/supaero-aircraft-design/FAST-GA/blob/main/docs/refs.bib